ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -20,6 +20,21 @@ const tankBusterOnParty = (data, matches) => {
   return data.party.inParty(data.target);
 };
 
+const getFacing = (combatant) => {
+  // Snap heading to closest card/intercard (aka PI/4).  N = PI, E = PI/2.
+  const facing = Math.round(combatant.Heading * 4 / Math.PI) / 4 * Math.PI;
```

Comment:
Ah, I see this came from e9s, probably.  I think it's a little weird floating point-wise to multiply by Math.PI here and then use Math.PI below.  This is especially true when north is both `Math.PI` and `-Math.PI`.

I think this might be more robust to do something like:
```
// 0 = N, 1 = E, 2 = S, 3 = W
const facing = (2 - Math.round(combatant.Heading * 4 / Math.PI) / 2) % 4;
```

Reply:
Ah, I never worked with the heading before. Logs were showing old data and I never tested out which headings map to which cardinal. I guess sometimes it's negative sometimes its positive pi radians. This works out when I test for the four cardinals. My original values were wrong then.

Reply:
You can use the cactbot test overlay to see your character heading, if you need an easy example.